packages/devtools/src/index.ts

```diff
@@ -122,12 +122,46 @@ export default class DevTools {
         return instance.sessionId
     }
 
-    /**
-     * allows user to attach to existing sessions
-     */
-    /* istanbul ignore next */
-    static attachToSession () {
-        throw new Error('not yet implemented')
+    static async attachToSession (options: AttachOptions, modifier?: Function, userPrototype = {}, customCommandWrapper?: Function) {
+        const browser = await launch(options.capabilities as ExtendedCapabilities)
```

Comment:
Why do we launch a browser here? Isn't the idea to attach to an existing browser? So you would need to use [`puppeteer.connect`](https://pptr.dev/#?product=Puppeteer&version=v8.0.0&show=api-puppeteerconnectoptions) using the options that you get.